packages/spark-core/src/spark-core.js

```diff
@@ -210,12 +210,6 @@ const SparkCore = AmpState.extend({
   logout(...args) {
     return Promise.resolve()
       .then(() => {
-        if (this.mercury) {
-          return this.mercury.disconnect();
-        }
-        return Promise.resolve();
-      })
-      .then(() => {
```

Comment:
That sounds like a web client bug. this change would cause nodejs apps to never disconnect from mercury, even after calling logout(). mercury emits several different offline events. the web client is probably listening to the wrong one.